cocotb/share/lib/simulator/simulatormodule.cpp

```diff
@@ -926,133 +926,152 @@ static PyMethodDef gpi_sim_hdl_methods[] = {
         (PyCFunction)get_signal_val_long, METH_NOARGS, PyDoc_STR(
             "get_signal_val_long($self)\n"
             "--\n\n"
-            "Get the value of a signal as a long"
+            "get_signal_val_long(self) -> int\n"
+            "Get the value of a signal as an integer."
         )
     },
     {"get_signal_val_str",
         (PyCFunction)get_signal_val_str, METH_NOARGS, PyDoc_STR(
             "get_signal_val_str($self)\n"
             "--\n\n"
-            "Get the value of a signal as an ASCII string"
+            "get_signal_val_str(self) -> str\n"
```

Comment:
Return annotations do not work here, sadly.

Reply:
?

Reply:
Nevermind, I misread this

Reply:
I do wish I could merge it with the text signature line, but that doesn't seem to render in sphinx correctly.

Reply:
That's what I meant - inspect.signature cannot parse return annotations in `__text_signature__`